docs/automation_host_prereqs.md

```diff
@@ -21,10 +21,10 @@ Install the following packages on the automation host. Select the appropriate in
 
 ### Terraform
 
-**Terraform >= 0.13.0**: Please refer to the [link](https://learn.hashicorp.com/terraform/getting-started/install.html) for instructions on installing Terraform. For validating the version run `terraform version` command after install.
+**Terraform >= 1.0.0**: Please refer to the [link](https://learn.hashicorp.com/terraform/getting-started/install.html) for instructions on installing Terraform. For validating the version run `terraform version` command after install.
 
 Install Terraform and providers for Power environment:
-1. Download the Terraform binary version 0.13.6 from https://www.power-devops.com/terraform and install it to /usr/local/bin.
+1. Download the Terraform binary version 1.0.8 from https://www.power-devops.com/terraform and install it to /usr/local/bin.
```

Comment:
`Download and install the latest (>= 1.0.0) Terraform binary for Linux/ppc64le from https://www.power-devops.com/terraform`